src/core/nodeOps.ts

```diff
@@ -337,10 +283,10 @@ export const nodeOps: () => RendererOptions<TresObject, TresObject | null> = ()
     patchProp,
     parentNode,
     createText: () => noop('createText'),
-    createComment,
+    createComment: () => noop('createComment'),
```

Comment:
Looks like this PR accidentally reverts #717 

Can we include those changes?

Reply:
For sure. Thanks!